make-resource.py

```diff
@@ -26,7 +25,7 @@ def allFiles(glob):
 if len(args.output) > 0 and not args.output.endswith('.qbtheme'):
     args.output += '.qbtheme'
 if not len(args.output) or os.path.exists(args.output):
-    if input('"{}" already exists, Do you want to overwrite?(y,n)'.format(args.output)) == 'n':
+    if input('"{}" already exists. Do you want to overwrite? (y,n) '.format(args.output)) == 'n':
```

Comment:
This is an old way to format strings and, with Python 3.5, we should try to favour f-strings as it brings consistency with other high-level languages.
```suggestion
    if input(f"{args.output} already exists. Do you want to overwrite? (y,n) ") == 'n':
```